src/modules/hmis/components/ClientDobAge.tsx

```diff
@@ -2,37 +2,52 @@ import { Stack, Typography, TypographyProps } from '@mui/material';
 import { ReactNode } from 'react';
 
 import ClickToShow from '@/components/elements/ClickToShow';
-import { age, dob } from '@/modules/hmis/hmisUtil';
+import { dob } from '@/modules/hmis/hmisUtil';
+import { useHasClientPermissions } from '@/modules/permissions/useHasPermissionsHooks';
 import { ClientIdentificationFieldsFragment } from '@/types/gqlTypes';
 
 interface Props {
   client: ClientIdentificationFieldsFragment;
   noValue?: ReactNode;
   variant?: TypographyProps['variant'];
   reveal?: boolean;
+  onlyAge?: boolean;
 }
 
 const ClientDobAge = ({
   client,
   noValue,
   reveal,
   variant = 'body2',
+  onlyAge,
 }: Props) => {
-  if (!client.dob) return <>{noValue}</> || null;
+  if (!client.dob && !client.age) return <>{noValue}</> || null;
```

Comment:
This will mess up if the age is zero. I have been using `isNil`.

Reply:
Fixed